auction9-backend/serverless.yml

```diff
@@ -110,3 +110,12 @@ functions:
           path: myauctions/{id}
           method: put
           cors: true
+
+  # creates new bid
+  postNewBid:
+    handler: handler.postNewBid
+    events:
+      - http:
+          path: auctions/{id}/new
```

Comment:
REST API URLs should reflect resources being operated with. In that sense better URL would be `auctions/{id}/bids`

Reply:
Changed to 'bids'